### ISSUE TITLE
Support for application/json+fhir

### ISSUE THREAD

**Issue by steveswinsburg** [mmattozzi/cocoa-rest-client#127]
I have a service which returns `application/json+fhir` as the content type. This does not pretty print, presumably because there is no support for that specific content type. The format is pretty much identical to `application/json` - it would be great if the pretty printer could parse that format.

**Comment by mmattozzi** [comment]
The app actually assumes any content type that ends with `+json` should be pretty printed as JSON (in addition to `application/json`). I didn't think about covering the reverse of that, `json+*`. Seems like there is some discussion about this WRT json+fhir here: https://github.com/jamesagnew/hapi-fhir/issues/445 

Should be easy enough to add however.

**Comment by mmattozzi** [comment]
Will add this to the next release.

**Comment by mmattozzi** [comment]
Fixed in https://github.com/mmattozzi/cocoa-rest-client/releases/tag/1.3.17

**Comment by steveswinsburg** [comment]
Thanks for fixing this ⭐️. In the 'Check For Update' option it says 'Cocoa Rest Client 1.3.15 is currently the newest version available.' I can manually update but is there some service that needs to get the new version number for the auto update?

**Comment by mmattozzi** [comment]
Between version 1.3.15 and 1.3.16, I bumped the required OS from 10.10.3 to the latest update of El Capitan, 10.11.6. The build target is still just `10.11` in the code, but I don't have the capability of testing at anything lower than `10.11.6` so I just set that as the requirement. By the way, the next official release will be Sierra only (looking ahead to 10.13 probably coming out this fall, I don't want to ever support more than two versions of macOS).
